app/views/playlists/index.html.erb

```diff
@@ -81,13 +81,14 @@ Unless required by applicable law or agreed to in writing, software distributed
     $('#Playlists').DataTable({
       processing: true,
       serverSide: true,
+        dom:'<"dataTableToolsTop"if><"dataTableBody"t><"dataTableToolsBottom"rlp>',
+      autoWidth:true,
       ajax: {
         url: '/playlists/paged_index',
         type: 'POST'
       },
       order: [ <%= "[#{session[:playlist_sort][0]}, '#{session[:playlist_sort][1]}']".html_safe if session[:playlist_sort] %>
       ],
-      "sDom": '<"playlists_top"i<"playlist_filter_container"f>>tr<"playlists_bottom"lp>',
```

Comment:
If removing the 'playlist_filter_container' class here, also remove it from css. Same with timeline_filter_container.